### PR TITLE
Portenta H7 Datasaheet: Fix typo.

### PR DESCRIPTION
**Pull request by iabdalkader**
## What This PR Changes
- Fix typo.

### CODE REVIEW

**Review by marqdevx — APPROVED** [review]
LGTM

**Comment by marqdevx** [comment]
@iabdalkader Sign the CLA and feel free to merge

**Comment by iabdalkader** [comment]
@marqdevx I can't merge this, I don't have write access. Please merge it or duplicate the fix

**Pull request merged**
